docs/specs/prw/remote_write_spec_2_0.md

```diff
@@ -249,13 +246,15 @@ message Request {
 
 // TimeSeries represents a single series.
 message TimeSeries {
+  reserved 6;
```

Comment:
nit: why not put it in order ? I'd move to the end

Reply:
This is for brevity. This is just shorter snippet of proto.